### ISSUE TITLE
Unable to get 4.2.2.0-RELEASE jar, Maven repository does not work and no jar attached

### ISSUE THREAD

**Issue by paulspeijers** [DICE-UNC/jargon#298]
Dear iRODS contributor,

When I read the following page it says a Maven repository is available:
https://github.com/DICE-UNC/jargon/releases/tag/4.2.2.0-RELEASE

Repository:
https://raw.github.com/DICE-UNC/DICE-Maven/master/releases

However when I open that URL I get a 404 not found.

When I resortied to adding jars to our Maven repostory manually I noticed that no jar has been included for 4.2.2.0-RELEASE. A jar is available for 4.2.0.0-RELEASE.

I found something looking like a Maven repository at the following URL:
https://github.com/DICE-UNC/DICE-Maven/tree/master/releases

However, there we don't have a valid Maven repository layout, neither a raw URL is available to use.

Bottom line:
Could you please setup a Maven repository I could use?
And it would also be good when a jar for 4.2.2.0-RELEASE is attached to Github.

Thanks in advance,

Cheers, Paul

**Comment by michael-conway** [comment]
The Maven artifacts are available in GitHub...e.g.
https://github.com/DICE-UNC/DICE-Maven/blob/master/releases/org/irods/jargon/jargon-core/4.0.2.2-RELEASE/jargon-core-4.0.2.2-RELEASE.jar

So if you add these to your pom you should be OK, but let me know if you
still have issues getting the artifacts!

<repositories>
<repository>
<id>dice.repository snaps</id>
<name>dice.repository.snapshots</name>
<url>https://raw.github.com/DICE-UNC/DICE-Maven/master/snapshots</url>
<releases>
<enabled>true</enabled>
</releases>
<snapshots>
<enabled>true</enabled>
<updatePolicy>always</updatePolicy>
<checksumPolicy>warn</checksumPolicy>
</snapshots>
</repository>
<repository>
<id>dice.repository</id>
<name>dice.repository</name>
<url>https://raw.github.com/DICE-UNC/DICE-Maven/master/releases</url>
<releases>
<enabled>true</enabled>
</releases>
<snapshots>
<enabled>false</enabled>
<updatePolicy>always</updatePolicy>
<checksumPolicy>warn</checksumPolicy>
</snapshots>
</repository></repositories>

**Comment by paulspeijers** [comment]
Hi Michael, thanks for your reply and good to get the location of the jar file. However, for me, the snapshots and releases repository really don't work. I get a 404 for both.

**Comment by paulspeijers** [comment]
After some testing I find out what the problem is, the URL is incorrect. I think it is based on the old GitHub urls, but should now be:
https://github.com/DICE-UNC/DICE-Maven/raw/master/releases

Could you confirm this, and if so change your documentation?

**Comment by michael-conway** [comment]
Gotcha thanks! I'll take a look at the docs, I'm putting together a release for the UGM
